pkg/cli/admin/catalog/build.go

```diff
@@ -43,8 +43,12 @@ type BuildImageOptions struct {
 }
 
 func NewBuildImageOptions(streams genericclioptions.IOStreams) *BuildImageOptions {
+	// Adjust default build opts to use an OpenShift base "from" image
+	defaults := appregistry.DefaultAppregistryBuildOptions()
+	defaults.From = "quay.io/openshift/origin-operator-registry:latest"
```

Comment:
should be a constant at the top?

Reply:
Sure, I'll go ahead and update that.

It seems like `e2e-aws-upgrade is consistently failing -- think it might be related?